workshop/content/30-hello-cdk/100-cleanup.md

```diff
@@ -0,0 +1,58 @@
++++
+title = "Cleanup sample"
+weight = 100
++++
+
+## Delete the sample code from your stack
+
+Open `bin/cdk-workshop.ts` and remove resources from your stack and the relevant
```

Comment:
This is not clear enough as to what you mean by resources. Suggest something like:
"The sample CDK project deploys an SQS queue, and an SNS topic. We're not going to use them in our project, so remove them from your CdkWorkshopStack constructor function."

Reply:
Fixed